dashboard/config/routes.rb

```diff
@@ -805,6 +805,8 @@
 
   post '/i18n/track_string_usage', action: :track_string_usage, controller: :i18n
 
+  get '/javabuilder/access_token', to: 'java_builder_sessions#get_access_token'
```

Comment:
Would it make sense to let this route automatically to `index` or `new`?

Reply:
good question--my concern with using one of those is it's not super clear what the function does in that case (get an access token).